ui/src/components/system/Dashboard.vue

```diff
@@ -907,9 +955,22 @@ export default {
               available_bytes: success.status.memory.SwapTotal
             }
           };
+          
+          context.system.disk = {
+            root: {
+              used_bytes: success.status.disk.root.used,
+              available_bytes: success.status.disk.root.free,
+              total_bytes: success.status.disk.root.total
```

Comment:
you can use only `used_bytes` and `available_bytes`. because the total is automatically calculated with the chart.

```js
used_bytes:
    success.status.disk.root.total -
    success.status.disk.root.free,
available_bytes: success.status.disk.root.total or success.status.disk.root.available (depends on your API)
```

Reply:
I used `total_bytes` for size span on the right bottom of the box. If you want I can remove `total_bytes` and perform a sum of used and available.
I didn't make a subtraction of `total` and `free` beacuse api can return also used disk space.

Reply:
ok, leave it like this.